workflow/rocoto/gfs_tasks.py

```diff
@@ -935,7 +935,10 @@ def _get_ufs_postproc_grps(cdump, config, component='atmos'):
         # Ocean/Ice components do not have a HF output option like the atmosphere
         if component in ['ocean', 'ice']:
             local_config['FHMAX_HF_GFS'] = config['FHMAX_GFS']
-            local_config['FHOUT_HF_GFS'] = config['FHOUT_GFS']
+            local_config['FHOUT_HF_GFS'] = config['FHOUT_OCNICE_GFS']
+            local_config['FHOUT_GFS']=config['FHOUT_OCNICE_GFS']
+            local_config['FHOUT_GFS']=config['FHOUT_OCNICE_GFS']
+            local_config['FHOUT']=config['FHOUT_OCNICE']
```

Comment:
```suggestion
            local_config['FHOUT'] = config['FHOUT_OCNICE']
```
Fix pycodestyle error